src/main/java/interface_adapter/student_home/StudentHomeController.java

```diff
@@ -0,0 +1,38 @@
+package interface_adapter.student_home;
+
+import use_case.student_homepage.StudentHomeInputBoundary;
+
+/**
+ * Controller for the StudentHome view.
+ */
+public class StudentHomeController {
+
+    private final StudentHomeInputBoundary studentHomeUseCaseInteractor;
+
+    public StudentHomeController(StudentHomeInputBoundary studentHomeUseCaseInteractor) {
+        this.studentHomeUseCaseInteractor = studentHomeUseCaseInteractor;
+    }
+
+    /**
+     * Executes the Controller.
+     * @param query the user input for the search field.
+     * @param email the current users email.
+     */
+    public void execute(String query, String email) {
```

Comment:
More clarification on the use case this does would be nice

Reply:
this was the remnant of my old structure, not needed anymore. removed the execute function.